go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/containers/common
 
 // Warning: Ensure the "go" and "toolchain" versions match exactly to prevent unwanted auto-updates
 
-go 1.22.8
+go 1.23.0
```

Comment:
Not sure what your timeline here is but just to be clear:
We are not yet ready to move to go 1.23 as we still use f40 in podman CI. We have planned to switch soon once the f42 beta is out.

Reply:
My timeline is runc 1.3 release which is end of April. It looks like it is more or less in line with F42 beta release / F40 EOL.

Reply:
We can actually do it earlier, as there's no runc 1.3 dependency